src/components/Asset/AssetContent/index.tsx

```diff
@@ -74,16 +74,42 @@ export default function AssetContent({
         </div>
 
         <div className={styles.actions}>
-          <AssetActions asset={asset} />
+          {!asset.accessDetails ? (
+            <p>Loading access details...</p>
+          ) : (
+            <>
+              {selectedService === -1 ? (
```

Comment:
what is -1?

Reply:
I updated it to undefined. It is a state when you don't have any service selected